WebRoot/templates/prefs/Pages.template

```diff
@@ -1091,8 +1096,8 @@ and the container element is replaced with that control.
 						<tr><td colspan=2></td></tr>
 					<$ } $>
 					<tr>
-						<td class='ZOptionsLabel'><$=ZmMsg.delegatesLabel$></td>
-						<td style='text-align:left'><$=ZmMsg.delegateRightsPrompt$></td></tr>
+						<td class='ZOptionsLabel ZOptionsSubHeadingLabel'><$=ZmMsg.delegatesLabel$></td>
+						<td style='text-align:left' class='ZOptionsSubHeadingContent'><$=ZmMsg.delegateRightsPrompt$></td></tr>
```

Comment:
Good catch. Actually this is not at all required. Removed here: 170445c09d8435d39607eedcd49019ba84a85152